### DISCUSSION TITLE
can not use codespace

### DISCUSSION THREAD

**Discussion by Faisal-Shumon** [community/community#27329] [Codespaces]
i can not use my codespace for cs50. name of codespace ""scaling space train main""- "code50/104099764 "
just black screen came only and above notifying "setting up your codespace". please help.

**Comment by cwndrws** [comment]
Hi 👋 thanks for reaching out!

How did you create this codespace? Through the CS50 portal? Or through github.com? Would you mind adding a screenshot of this screen to your question? How long did you wait when on this screen?

**Reply by Faisal-Shumon** [reply]
Actually, 3 days ago i worked on this codespace and submit file. But from yesterday, getting just this problem. Black screen come and stuck. nothing come.

i used to cs50 portal to create codespace around 3 months ago.

please see the below screenshot.

![Github-solution](https://user-images.githubusercontent.com/104099764/181296985-2660f9a7-7ad4-4129-945e-9aa703330a34.JPG)

**Reply by cwndrws** [reply]
Could you try again now and let report back what you see? There was some stale data that we just resynced.

**Reply by Faisal-Shumon** [reply]
still same result, even after changing browser.

**Reply by cwndrws** [reply]
Are there any browser console errors? You can follow this to send me the log: https://docs.github.com/en/codespaces/troubleshooting/github-codespaces-logs#browser-console-logs. If it looks like there's sensitive information in the logs, you can email me the log. My email is probably pretty easy to guess. It's my username at the website we're currently on.

**Reply by Faisal-Shumon** [reply]
please see the screenshot of Errors.  After browsing codespace, this browser console log taken. 

![image](https://user-images.githubusercontent.com/104099764/181310383-28144cf8-6c8c-4cc1-8d1f-152d47447fab.png)

**Reply by Faisal-Shumon** [reply]
this log from another browser. please see the below screenshot.

![image](https://user-images.githubusercontent.com/104099764/181313319-23b08241-a5c1-4554-8fff-e193f0ab3205.png)
![image](https://user-images.githubusercontent.com/104099764/181313770-fdd59f34-e6cf-4d70-8a90-aa1a5ae4f532.png)

**Reply by cwndrws** [reply]
Are you behind a corporate firewall? Or using some sort of proxy? Or are you maybe using some non-standard certificate authority? it looks like your browser doesn't like the TLS cert for https://update.code.visualstudio.com/api/update/web-standalone/stable/latest. I'm able to get to that page fine, so it seems like something specific to your machine or network.

If you have VS Code desktop, it might work through that using the GitHub Codespaces extension rather than in the web.

**Reply by Faisal-Shumon** [reply]
Hello  Charlie Andrews, 
At first, i apologize. My desktop browser was in backdated version. After updating, Now codespace is working. 
Thanks for your excellent support.

**Reply by cwndrws** [reply]
You're welcome! I hope you're enjoying using Codespaces!

**Reply by Faisal-Shumon** [reply]
Hello Charlie Andrews,
i can not open my cs50 Visual studio in codespace. just screen showing starting or connecting.
can you help, please ?

**Reply by cwndrws** [reply]
Are there any errors in your browser's developer tools console? https://docs.github.com/en/codespaces/troubleshooting/github-codespaces-logs#browser-console-logs are the instructions for viewing these logs.

**Comment by Faisal-Shumon** [comment]
Hello Charlie Andrews,
i can not open  my cs50 Visual studio in codespace.    just screen showing starting or connecting.
can you help, please ?
